sktime/forecasting/base/_base.py

```diff
@@ -1998,7 +1998,9 @@ def _predict_interval(self, fh, X=None, coverage=0.90):
                 "This is likely a bug, please report, and/or set the flag to False."
             )
 
-        if implements_quantiles:
+        # we default to _predict_quantiles if that is implemented or _predict_proba
+        # since _predict_quantiles will default to _predict_proba if it is not
+        if implements_quantiles or implements_proba:
```

Comment:
I think that the check with `can_do_proba`in L1993 and the following `raise` should ensure that check in L2003 will always pass. So it can be skipped, and the following section can be de-indented.

Reply:
you are right - changed